docs/content/reference.md

```diff
@@ -0,0 +1,494 @@
+6. Populate the content as appropriate, using the [reference](#reference) to help.
+
+
+# Localization
+
+An experiment's YAML file is considered to contain the canonical localization for English, and is compiled into [an FTL file](https://testpilot.dev.mozaws.net/static/locales/en-US/experiments.ftl) with each build. That file is monitored by Pontoon, and strings are translated by volunteer localizers.
+
+If the content of a string changes sufficiently that translations of it should be invalidated:
+
+1. Make sure that this change needs to be done. We rely on volunteer localizers, 
+2. If at all possible, wait to make the change until the beginning of a sprint to give localizers time to translate the new string before the next deployment.
+3. Add a new field to the YAML titled `<fieldname>_l10nsuffix`, set to a short string contextualizing the change.
+
+For example: if we decide to rename the "Activity Streams" experiment to "Activity Stream", the following YAML:
+
+```yaml
+title: 'Activity Streams'
+```
+
+Would be changed to this:
+
+```yaml
+title: 'Activity Stream'
+title_l10nsuffix: 'singular'
+```
+
+This change will turn the `l10n-id` from `activitystreamTitle` to `activitystreamTitleSingular`, treating it as an entirely new string and invalidating all existing translations.
+
+This also works for nested properties:
+
+```yaml
+contributors:
+  -
+      display_name: 'Jared Hirsch'
+      title: 'Staff Engineer'
+      title_l10nsuffix: 'staff'
+      
+```
+
+
+# Reference
+
+The following YAML fields are available to content authors:
+
+
+## `id`
+
+A unique numeric ID for the experiment. This is being incremented sequentially, so check previous experiments to determine what this value should be. Required.
+
+```yaml
+id: 0
+```
+
+
+## `slug`
+
+A slugified version of the title, used to generate URLs. Required.
+
+```yaml
+slug: 'experiment-name'
+```
+
+
+## `title`
+
+The name of the experiment. Localized, required.
+
+```yaml
+title: 'Experiment Name'
+```
+
+
+## `subtitle`
+
+A subtitle for the experiment, generally used for attribution of an experiment to a partner. Localized, optional.
+
+```yaml
+subtitle: 'Powered by a yellow submarine'
+```
+
+![Example use of subtitle](img/subtitle.png)
+
+
+## `order`
+
+The order in which the experiment will appear. Lower numbers appear first. Required.
+
+```yaml
+order: 0
+```
+
+## `description`
+
+A short description of the experiment, used for search engines, social media shares, and in the experiment card. Localized, required, HTML not allowed.
+
+```yaml
+description: >
+  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsum cum ad deserunt
+  iusto possimus. Fugiat odit corrupti cumque.
+```
+
+![Example Twitter share with description](img/description.png)
+
+
+## `introduction`
+
+A short introductioni of the experiment, displayed on top of the experiment detail page. Required, localized, HTML required.
```

Comment:
@pdehaan i went ahead and merged this b/c i need to add to it in another PR...will fix the error